x/stake/keeper/slash.go

```diff
@@ -134,12 +143,12 @@ func (k Keeper) Unjail(ctx sdk.Context, pubkey crypto.PubKey) {
 }
 
 // set the jailed flag on a validator
-func (k Keeper) setJailed(ctx sdk.Context, pubkey crypto.PubKey, jailed bool) {
+func (k Keeper) setJailed(ctx sdk.Context, pubkey crypto.PubKey, isJailed bool) {
```

Comment:
(out of curiosity) why did you decide to rename?

Reply:
When I first read this it confused me a bit, had to read it twice before it clicked - so I figured it'd probably confuse others at some point too. I think the word jailed might be overloaded just a tiny bit - I figured `isJailed` is just a little bit more explicit. Very minor though.